### ISSUE TITLE
Conflict in input file reading

### ISSUE THREAD

**Issue by deb89** [Melkiades/CampaRi#27]
There is a conflict in the input file reading. For example in tutorial 11:

If I provide both `FMCSC_DCDFILE NBU.dcd` in the key file and `data_file = "NBU.dcd"` as parameter to run_campari() I get the following error:

`Error in run_campari(data_file = "NBU.dcd", key_file_input = "nbu_clustering.key",  : 
  We found both data_file and one of 'FMCSC_XTCFILE', 'FMCSC_DCDFILE', 'FMCSC_PDBFILE', 'FMCSC_NETCDFFILE' keywords. Please select only one of those options.`

I can remove the parameter `data_file = "NBU.dcd"` from run_campari() but I get the following warning:

`Warning messages:
1: In run_campari(key_file_input = "nbu_clustering.key", FMCSC_SEQFILE = "nbu.in",  :
  We found analysis keywords while no trj nor data_file was supplied. All the checks for the analysis inputs will be disabled. No NCMINER mode active.

2: In run_campari(key_file_input = "nbu_clustering.key", FMCSC_SEQFILE = "nbu.in",  :
  Even if FMCSC_PDB_FORMAT was used for FMCSC_DCDFILE no standard (4) was selected. It will be defaulted to 4.`

So for the moment the only clean way to run it appears to be by removing the  FMCSC_DCDFILE keyword from the key file.

**Comment by Melkiades** [comment]
Yes that is expected. In the analysis, you can use the automatic run_campari() selection of the needed keywords for a specific file extension (which is advised as it would select also the right bin executable) OR you put it manually in the keyfile or as "FMCSC_*" in the function call.

**Comment by Melkiades** [comment]
The warning simply says that the checks on the input files are turned off because you used the key file instead of the function to indicate the trajectory to analyse. In this case, there is no check in place and, therefore, it is not possible to select the camp_ncminer executable in case of netcdf or ascii file (tsv, csv).
